sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -160,3 +161,23 @@ export function toBuffer(input: any): Buffer {
   return result;
 }
 
+/**
+ * Helper function to retrieve active receiver name, if it exists.
+ *
```

Comment:
Please add the tag `@internal` here otherwise, the function gets picked up by our doc generation process. For example: https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/index?view=azure-node-preview#functions

Reply:
Wouldn't all of the methods in `utils.ts` need to be marked as internal?